src/App.jsx

```diff
@@ -1,21 +1,25 @@
-import './App.css'
-import TodoList from './TodoList'
-import AddTodoForm from './AddTodoForm'
-import { useState } from 'react'
+import './App.css';
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
+import { useState } from 'react';
 
 function App() {
-  const [newTodo, setNewTodo] = useState('')
+  const [newTodo, setNewTodo] = useState('');
```

Comment:
Is this still needed?

Reply:
> Is this still needed?

No. I removed it and the line that used it. The state is not updated anywhere for this var.